src/build.props

```diff
@@ -37,13 +28,49 @@
   </PropertyGroup>
 
   <PropertyGroup Label="Build">
+    <OutputSubDir>$(Platform)_$(Configuration)</OutputSubDir>
```

Comment:
>OutputSubDir [](start = 5, length = 12)

All the added lines come from the now-removed `build.common.props`. I interleaved its contents in the appropriate places.